pkgs/data/fonts/source-han/default.nix

```
@@ -1,27 +1,28 @@
 { stdenvNoCC
 , lib
 , fetchzip
-, fetchurl
 }:
 
 let
-  makePackage = { family, description, rev, sha256 }: let
-    Family =
+  makePackage = {
+    family,
+    description,
+    rev,
+    sha256,
+    postFetch ? ''
+      install -m444 -Dt $out/share/fonts/opentype/source-han-${family} $downloadedFile
+    '',
+    zip ? ""
```

```suggestion
  makePackage = { 
    family
    , description
    , rev
    , sha256
    ,  postFetch ? ''
        install -m444 -Dt $out/share/fonts/opentype/source-han-${family} $downloadedFile
      ''
    , zip ? ""
```
or just use nixpkgs-fmt which probably has a better idea what to do.

---

Thanks! I tested with nixfmt but the result looks worse. nixpkgs -fmt works better for me.